extension/src/popup/components/sendPayment/SendSettings/Slippage/index.tsx

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Field, Form, Formik, FieldProps } from "formik";
+
+import {
+  Button,
+  Input,
+  Icon,
+  TextLink,
+  Card,
+  DetailsTooltip,
+} from "@stellar/design-system";
+
+import {
+  transactionDataSelector,
+  saveAllowedSlippage,
+} from "popup/ducks/transactionSubmission";
+import { navigateTo } from "popup/helpers/navigate";
+import { ROUTES } from "popup/constants/routes";
+import { PopupWrapper } from "popup/basics/PopupWrapper";
+
+import "./styles.scss";
+
+const defaultSlippage = "1";
+
+export const SendSettingsSlippage = () => {
+  const dispatch = useDispatch();
+  const { allowedSlippage } = useSelector(transactionDataSelector);
+
+  let presetSlippage = "";
+  let customSlippage = "";
+  if (["1", "2", "3"].includes(allowedSlippage)) {
+    presetSlippage = allowedSlippage;
+  } else {
+    customSlippage = allowedSlippage;
```

Comment:
Bit of a bummer that we have to use 2 vars here, but i couldn't think of a better way to do this either

Reply:
ya  found it easier to separate them for the UI :/